migrations/V1__my_first_migration.sql

```diff
@@ -0,0 +1,4 @@
+create table PERSON (
```

Comment:
This is going to actually create the `person` table. Do we want to instead create one for our actual application? Optionally, we don't need a migration, and can work on the ticket for an `intake` table migration after.

Reply:
Is there a foreign key relationship between those two tables, or some other dependency that would necessitate creating them in a particular order? What goes in the table for the application?

Reply:
Agree that this migration can be nixed and then merged.